sentry-android-core/src/main/java/io/sentry/android/core/SentryLogcatAdapter.java

```diff
@@ -44,73 +48,104 @@ private static void addAsBreadcrumb(
     Sentry.addBreadcrumb(breadcrumb);
   }
 
+  private static void addAsLog(
```

Comment:
Is there already a way of setting a min level for logs coming from logcat so we don't send tons of data to Sentry?

Reply:
this works only with the gradle plugin, and we have an option to [set the minimum level there](https://docs.sentry.io/platforms/android/integrations/logcat/#configure)
i'll add a remainder in the changelog about it, and when we'll write the docs, we'll keep it in mind